tests/js/object-tests.js

```diff
@@ -567,6 +568,7 @@ module.exports = {
 
         obj.addListener((obj, changes) => {
             calls++;
+            console.log(`FISK ${calls}: ${JSON.stringify(changes)}`);
```

Comment:
I don't know if it's the source of problems, but one thing I notice about this test is that it seems like `obj` could be garbage collected immediately after the call to `realm.delete()` which would result in the notifier being destroyed.